pkg/arm/data/node-startup.sh

```diff
@@ -25,7 +25,9 @@ while [[ $(hostname -d) == "" ]]; do sleep 1; done
 while ! docker pull {{ .Config.Images.Startup }}; do
   sleep 1
 done
-docker run --privileged --rm --network host -v /:/host:z {{ .Config.Images.Startup }} /host$PWD/worker-startup
+set +x
+docker run --privileged --rm --network host -v /:/host:z {{ .Config.Images.Startup }} '{{ .Config.WorkerStartupSASURI }}'
```

Comment:
can we move this args into the container? I dont like this way just because it will show in `ps aux`. so could be logged. Or it might be just me and my overthinking.

Reply:
overthinking...

Reply:
you're right, should be an environment variable

Reply:
suggest to fix up in a subsequent patch set

Reply:
fixed